packages/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -65,19 +65,16 @@
 		</span>
 	{:else}
 		<select
-			disabled={hasQuery && !$query.loaded}
-			on:change={(e) => ($inputs[name] = e.currentTarget.value)}
+			disabled={hasQuery && !$query?.loaded}
```

Comment:
```suggestion
			disabled={hasQuery && !$query.loaded}
```
I don't think it should be possible for `$query` to be undefined if `hasQuery`?